src/main/java/org/jabref/model/search/query/SearchQuery.java

```diff
@@ -32,6 +36,7 @@ public SearchQuery(String searchExpression, EnumSet<SearchFlags> searchFlags) {
             this.context = getStartContext(searchExpression);
             isValidExpression = true;
         } catch (ParseCancellationException e) {
+            LOGGER.error("Search query Parsing error", e.getCause());
```

Comment:
```suggestion
            LOGGER.error("Search query Parsing error", e);
```

Reply:
Nope, this was deliberate, because the cause here holds more relevant info and prevents the stacktrace from getting too long

Reply:
Java comment? Because someone else could stumble on it?